airflow/ui/src/components/DocumentationModal.tsx

```diff
@@ -23,14 +23,21 @@ import Markdown from "react-markdown";
 
 import { Button, Dialog } from "src/components/ui";
 
-const DagDocumentation = ({ docMd }: { readonly docMd: string }) => {
+const DocumentationModal = ({
+  docMd,
+  isDag = false,
```

Comment:
Instead of making it boolean... atm it is used only in task and dag... would it make sense to rather pass a `readonly docType: string` as parameter and use this as `docType` directly?

Reply:
Makes sense, thanks.

Reply:
No strong opinion on that. (at least we don't have the confusion that `isDag=False` and it's a new entity that doesn't yet exist that is also not a task, more extensible, but really nit I don't mind)